src/index.ts

```diff
@@ -0,0 +1,677 @@
+          const packageManagerFiles =
+            (accumulator[file.packageManager] as Array<IManifestFile> | undefined) ?? [];
+          return {
+            ...accumulator,
+            [file.packageManager]: packageManagerFiles.concat(file),
+          };
+        },
+        {}
+      );
+
+      // note: assuming failure until proven otherwise
+      let allUploadsFailed = true;
+
+      for (const [packageManager, files] of Object.entries(manifestsByPackageManager)) {
+        try {
+          const manifestUploadResponse = await this.uploadManifestFilesByPackageManager({
+            apiClient: soosAnalysisApiClient,
+            clientId: this.args.clientId,
+            projectHash,
+            branchHash,
+            analysisId,
+            manifestFiles: files,
+          });
+
+          soosLogger.info(
+            `${packageManager} Manifest Files: \n`,
+            `  ${manifestUploadResponse.message} \n`,
+            manifestUploadResponse.manifests
+              ?.map((m) => `  ${m.name}: ${m.statusMessage}`)
+              .join("\n")
+          );
+
+          allUploadsFailed = false;
+        } catch (e: unknown) {
+          // NOTE: we continue on to the other package managers, but log it as a warning in the log
+          const message = e instanceof Error ? e.message : (e as string);
+          soosLogger.warn(message);
+        }
+      }
+
+      if (allUploadsFailed) {
+        await soosAnalysisApiClient.updateScanStatus({
+          clientId: this.args.clientId,
+          projectHash,
+          branchHash,
+          scanType: ScanType.SCA,
+          scanId: analysisId,
+          status: ScanStatus.Incomplete,
+          message: `Error uploading manifests.`,
+        });
+        soosLogger.error("Error uploading manifests.");
+        return;
+      }
+
+      soosLogger.logLineSeparator();
+      soosLogger.info("Starting SCA Analysis scan");
+      await soosAnalysisApiClient.startScan({
+        clientId: this.args.clientId,
+        projectHash,
+        analysisId: analysisId,
+      });
+      soosLogger.info(
+        `Analysis scan started successfully, to see the results visit: ${result.scanUrl}`
+      );
+
+      const scanStatus = await this.waitForScanToFinish({
+        apiClient: soosAnalysisApiClient,
+        scanStatusUrl: result.scanStatusUrl,
+        scanUrl: result.scanUrl,
+      });
+
+      if (this.args.outputFormat !== undefined) {
+        soosLogger.info(`Generating SARIF report  ${this.args.projectName}...`);
+        const output = await soosAnalysisApiClient.getFormattedScanResult({
+          clientId: this.args.clientId,
+          projectHash,
+          branchHash,
+          scanType: ScanType.SCA,
+          scanId: analysisId,
+          outputFormat: this.args.outputFormat,
+        });
+        if (output) {
+          soosLogger.info(`SARIF report generated successfully.`);
+          soosLogger.info(`Output ('${this.args.outputFormat}' format):`);
```

Comment:
since we say it is SARIF in the previous - probably can remove the format message - or you can use the format in the success message:
```suggestion
          soosLogger.info(`SARIF report generated successfully.`);
```
or
```suggestion
          soosLogger.info(`${this.args.outputFormat} report generated successfully.`);
```